app/assets/javascripts/pretty-text/inline-oneboxer.js.es6

```diff
@@ -1,5 +1,7 @@
 let _cache = {};
 
+export const INLINE_ONEBOX_CSS_CLASS = "inline-onebox-loading";
```

Comment:
I'm always annoyed by CONSTANTS existing in both ruby and js world. Could we make this an erb file and have this value coming from a constant in `lib/cooked_post_processor.rb` ?

This way we are sure we have only one place to change.